contents/ruby/Rails.md

```diff
@@ -0,0 +1,53 @@
+<frontmatter>
+  title: Introduction to Ruby on Rails
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Ruby on Rails
+
+Authors: [Chattoraj Ayush](https://github.com/AyushChatto)
+
+## Ruby on Rails overview
+
+**Ruby on Rails** is an **easy-to-learn** framework that allows you to develop powerful web applications **quickly** and with as little effort as possible. It was developed in the early 2000s by Danish Programmer David Heinemeier Hansson, or 'DHH'. 
+
+Ruby on Rails, or simply Rails, gained widespread popularity in the 2000s, and continues to be one of the most popular web-frameworks even to this day, because of how quickly it allows an easily scalable minimum viable product to be built, making it the perfect choice for startups. This is owed in large part to the Rails Doctrines of **Optimising for Programmer Happiness** and **Convention over Configuration**.
+
+> ... I created Rails for me. To make me smile, first and foremost. Its utility was to many degrees subservient to its ability to make me enjoy my life more. To enrich my daily toil of wrangling requirements and requests for web information systems.
+>
+> ... One of the early productivity mottos of Rails went: “You’re not a beautiful and unique snowflake”. It postulated that by giving up vain individuality, you can leapfrog the toils of mundane decisions, and make faster progress in areas that really matter.
+>
+> *David Heinemeier Hansson,* __The Rails Doctrine__
+
+Many popular and very technically mature websites continue to use Rails. These include: GitHub, Airbnb, Twitter, Hulu, Shopify, and Twitch, among others.
+
+## Getting Started
+
+The official website for [Rails](https://rubyonrails.org) will have all the information needed to initialize your first Rails application.
+
+To install Rails on your computer, you will first need to install [Ruby](https://www.ruby-lang.org/en/). After this, you simply need to run:
+
+```rb
+gem install rails
+```
+gem is a package manager for Ruby that allows you to install Rails. Once you're done with the setup, you can start developing applications right away. While the [Rails Guides](https://guides.rubyonrails.org/) are the best possible place to learn everything about Rails, [Go Rails](https://gorails.com/) also has great screencast tutorials that really hold your hand through the multitude of features Rails has, and can be used by more visual learners.
```

Comment:
> gem is a package manager for Ruby that allows you to install Rails

I think there's a greater opportunity here to follow up with how there's a thriving Ruby community and that there are many well developed Ruby gems out there that you can easily plug in to speed up the development of your app. (Maybe mention a few established gems like `devise`)

---

> While the [Rails Guides](https://guides.rubyonrails.org/) are the best possible place to learn everything about Rails, [Go Rails](https://gorails.com/) also has great screencast tutorials...

This part is probably more suited for the `Further Reading` section.

Reply:
> how there's a thriving Ruby community and that there are many well developed Ruby gems

Great Idea :D 

> This part is probably more suited for the `Further Reading` section.

I meant that as the starting point for Rails, but I'll cross that bridge when I get there, since I'll move it further down.